network-api/networkapi/wagtailpages/factory/homepage_features.py

```diff
@@ -60,4 +61,12 @@ def generate(seed):
         for i in range(NUM_HIGHLIGHTS)
     ]
 
+    home_page.spotlight_posts = [
+        HomepageSpotlightPosts.objects.create(
+            page=home_page,
+            blog=BlogPage.objects.order_by("?").first()
+        )
+        for i in range(3)
```

Comment:
So that this follows the existing code in this file -- let's store `3` in a variable `NUM_SPOTLIGHT_POSTS` and use that variable here

```py
NUM_BLOGS = 4
NUM_HIGHLIGHTS = 5
NUM_SPOTLIGHT_POSTS = 3 // add this line
```